toolkit/dump_tasks.php

```diff
@@ -44,27 +44,32 @@
 		exit - 1;
 	}
 
+	echo "iTop: $sITopUrl\n";
 	if (empty($aResult['objects'])) {
-		echo "There is no SynchroDataSource defined on the iTop server ($sITopUrl).\n";
+		echo "   no SynchroDataSource defined\n";
 	} else {
-		switch (count($aResult['objects'])){
+		switch (count($aResult['objects'])) {
 			case 1:
-				echo "There is 1 SynchroDataSource defined on the iTop server ($sITopUrl):\n";
+				echo "   1 SynchroDataSource defined\n";
 				break;
 
 			default:
-				echo "There are ".count($aResult['objects'])." SynchroDataSource defined on the iTop server ($sITopUrl):\n";
+				echo '   '.count($aResult['objects'])." SynchroDataSource objects defined\n";
```

Comment:
```suggestion
				echo "   ".count($aResult['objects'])." SynchroDataSource objects defined\n";
```

Reply:
Why ? 
I tend to use `"` instead of `'` as string delimiters only when necessary.
Here in the second part of the string we have `\n` that make `"` delimiters mandatory, but in the first part we only have spaces so no need to use `"` right ?

Reply:
I tend to favor consistency in this kind of thing, especially as there is no performance difference.

Reply:
Ok
`"` are kind of dangerous as they are really different than `'`, but here the risk is very not significant.

Reply:
Could also use `PHP_EOL` to reduce the need for `"` ;-)